### PR TITLE
Adduserv4

### PR DESCRIPTION
**Pull request by kevdaci**
## Commit Message Guidelines
```
Header
Blank Line
Body
Blank Line
Footer

The header should look like:
<type>(<scope>): <subject>

The body should have any necessary detailed info about the commit:
An example, references as to where this idea came from, etc.

The footer should have all the issues tagged:
Fixes #123, Fixes #456

So a commit should look like:
feat(users): Add new Yahoo authentication

Yahoo authentication idea proposed by @codydaig
Example implementation in file.js

Fixes #82
```

* Types: 
  * feat - Features, Enhancements, and overall Improvements
  * fix - Fixes, Bugs, HotFixs, etc...
  * doc - Changes to the Documentation and doesn't actually touch any code.
* Scope:
  * The scope should be where the change took place.
  * Examples: users, core, config, articles
* Subject:
  * The subject line should be clear and concise as to what is being accomplished in the commit.
* General Rules:
  * No Line in the Commit message can be longer than 80 characters.
* Reference: [Angular Conventions](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md)

### CODE REVIEW

**Comment by markd315** [comment]
I checked this branch out and it doesn't fix this bug we're targeting. I'm not sure what it does fix, but when I create a new user as an admin and there's no dropdown for the signup page.

**Comment by markd315** [comment]
This branch also badly needs a rebase, it is very out of date with develop.

**Review by markd315 — CHANGES_REQUESTED** [review]
I checked this branch out and it doesn't fix this bug we're targeting. I'm not sure what it does fix, but when I create a new user as an admin the role is still wrong and there's no dropdown for the signup page for the other way of creating users.

This branch also badly needs a rebase, it is very out of date with develop.

**Comment by kevdaci** [comment]
@markd315  I am adding this feature because it was one of the stories that should have been added from the second sprint, but it was not added on time. I will edit and rebase it.

File: modules/users/client/config/users-admin.client.routes.js [hunk 1]
```diff
@@ -32,7 +32,7 @@
         url: '/add',
         templateUrl: '/modules/users/client/views/admin/add-user.client.view.html',
         controller: 'AddUserController',
-        controllerAs: 'vm'
+        controllerAs: 'vm',
```
Comment (RyanClementsHax) [inline modules/users/client/config/users-admin.client.routes.js:35]: unecessary comma

**Review by RyanClementsHax — CHANGES_REQUESTED** [review]
This seems exactly like another pull request we have open

**Pull request closed without merging**